### PR TITLE
Ofir/tfl 1370

### PR DESCRIPTION
**Pull request by OfirZinkerz**

### CODE REVIEW

**Comment by orenor10** [comment]
Merge with zinkerz/dev

File: Gruntfile.js [hunk 1]
```diff
@@ -258,25 +270,17 @@ module.exports = function (grunt) {
                     }
                 }]
             },
-            serve:{
```
Comment (assafshp) [inline Gruntfile.js]: are you sure it is not needed anymore?

File: demo/loginApp/demo.js [hunk 2]
```diff
@@ -1,41 +1,341 @@
...
-    .config(function ($translateProvider, $locationProvider, PromoCodeSrvProvider, LoginAppSrvProvider, AllEnvs) {
+    .config(function ($locationProvider, PromoCodeSrvProvider, AllEnvConfigSrvProvider, LoginAppSrvProvider) {
```
Comment (assafshp) [inline demo/loginApp/demo.js]: try to merge from zinkerz/dev or ask Kosta if it is ok?

File: dist/feedback/feedback.js [hunk 3]
```diff
@@ -12,18 +12,17 @@
             'znk.infra.user',
             'znk.infra.svgIcon'
         ])
-        .config([
-            'SvgIconSrvProvider',
-            function (SvgIconSrvProvider) {
+        .config(
+            ["SvgIconSrvProvider", function (SvgIconSrvProvider) {
+                'ngInject';
```
Comment (assafshp) [inline dist/feedback/feedback.js]: good !

File: dist/main.js [hunk 4]
```diff
@@ -8676,6 +8602,27 @@ angular.module('znk.infra-web-app.promoCode').run(['$templateCache', function($t
...
     angular.module('znk.infra-web-app.purchase')
+        .config(
+            ["SvgIconSrvProvider", function (SvgIconSrvProvider) {
+                'ngInject';
+                var svgMap = {
+                    'purchase-check-mark': 'components/purchase/svg/check-mark-icon.svg',
+                    'purchase-close-popup': 'components/purchase/svg/purchase-close-popup.svg',
+                    'purchase-popup-bullet-1-icon': 'components/purchase/svg/purchase-popup-bullet-1-icon.svg',
+                    'purchase-popup-bullet-2-icon': 'components/purchase/svg/purchase-popup-bullet-2-icon.svg',
+                    'purchase-popup-bullet-3-icon': 'components/purchase/svg/purchase-popup-bullet-3-icon.svg',
```
Comment (assafshp) [inline dist/main.js]: why we need it herer?

**Comment by assafshp** [comment]
fine by me, just merge with zinkerz/dev and lets do it

**Pull request merged**